### ISSUE TITLE
[bug] Current docker build does not work as expected

### ISSUE THREAD

**Issue by snailcatcher** [StevenWeathers/thunderdome-planning-poker#154]
Hi,
I get a few errors with the latest docker release (must be 1.6.0):

On first clean startup I get: (this is the full log, the exception comes halfway down)
```
Starting utility-hashing_db_1 ... done
Recreating utility-hashing_thunderdome_1 ... done
Attaching to utility-hashing_db_1, utility-hashing_thunderdome_1
db_1           | The files belonging to this database system will be owned by user "postgres".
db_1           | This user must also own the server process.
db_1           |
db_1           | The database cluster will be initialized with locale "en_US.utf8".
db_1           | The default database encoding has accordingly been set to "UTF8".
db_1           | The default text search configuration will be set to "english".
db_1           |
db_1           | Data page checksums are disabled.
db_1           |
thunderdome_1  | 2021/04/12 14:39:38 Thunderdome version 1.6.0
thunderdome_1  | 2021/04/12 14:39:38 dial tcp 172.22.0.2:5432: connect: connection refused
db_1           | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db_1           | creating subdirectories ... ok
db_1           | selecting dynamic shared memory implementation ... posix
db_1           | selecting default max_connections ... 100
db_1           | selecting default shared_buffers ... 128MB
db_1           | selecting default time zone ... Etc/UTC
db_1           | creating configuration files ... ok
utility-hashing_thunderdome_1 exited with code 1
Exception in thread Thread-7:
Traceback (most recent call last):
  File "docker\api\client.py", line 268, in _raise_for_status
  File "requests\models.py", line 941, in raise_for_status
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: http+docker://localnpipe/v1.41/containers/a4ce08f6ff668a78822eec2e1b65cba79eddd5755375a39dafd0b321978fabc1/attach?logs=0&stdout=1&stderr=1&stream=1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 950, in _bootstrap_inner
  File "threading.py", line 888, in run
  File "compose\cli\log_printer.py", line 224, in watch_events
  File "compose\container.py", line 202, in attach_log_stream
  File "compose\container.py", line 294, in attach
  File "docker\utils\decorators.py", line 19, in wrapped
  File "docker\api\container.py", line 62, in attach
  File "docker\api\client.py", line 407, in _read_from_socket
  File "docker\api\client.py", line 318, in _get_raw_response_socket
  File "docker\api\client.py", line 270, in _raise_for_status
  File "docker\errors.py", line 19, in create_api_error_from_http_exception
  File "requests\models.py", line 881, in json
  File "requests\models.py", line 829, in content
  File "requests\models.py", line 751, in generate
  File "urllib3\response.py", line 575, in stream
  File "urllib3\response.py", line 518, in read
  File "http\client.py", line 458, in read
  File "http\client.py", line 502, in readinto
  File "docker\transport\npipesocket.py", line 211, in readinto
  File "docker\transport\npipesocket.py", line 20, in wrapped
RuntimeError: Can not reuse socket after connection was closed.
db_1           | running bootstrap script ... ok
db_1           | performing post-bootstrap initialization ... ok
db_1           | syncing data to disk ... ok
db_1           |
db_1           |
db_1           | Success. You can now start the database server using:
db_1           |
db_1           |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db_1           |
db_1           | initdb: warning: enabling "trust" authentication for local connections
db_1           | You can change this by editing pg_hba.conf or using the option -A, or 
db_1           | --auth-local and --auth-host, the next time you run initdb.
db_1           | waiting for server to start....2021-04-12 14:39:47.371 UTC [48] LOG:  starting PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
db_1           | 2021-04-12 14:39:47.374 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1           | 2021-04-12 14:39:47.399 UTC [49] LOG:  database system was shut down at 2021-04-12 14:39:44 UTC
db_1           | 2021-04-12 14:39:47.426 UTC [48] LOG:  database system is ready to accept connections
db_1           |  done
db_1           | server started
db_1           | CREATE DATABASE
db_1           |
db_1           |
db_1           | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
db_1           |
db_1           | 2021-04-12 14:39:49.160 UTC [48] LOG:  received fast shutdown request      
db_1           | waiting for server to shut down....2021-04-12 14:39:49.164 UTC [48] LOG:  aborting any active transactions
db_1           | 2021-04-12 14:39:49.165 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
db_1           | 2021-04-12 14:39:49.165 UTC [50] LOG:  shutting down
db_1           | 2021-04-12 14:39:49.224 UTC [48] LOG:  database system is shut down
db_1           |  done
db_1           | server stopped
db_1           |
db_1           | PostgreSQL init process complete; ready for start up.
db_1           |
db_1           | 2021-04-12 14:39:49.294 UTC [1] LOG:  starting PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
db_1           | 2021-04-12 14:39:49.294 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db_1           | 2021-04-12 14:39:49.294 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db_1           | 2021-04-12 14:39:49.300 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db_1           | 2021-04-12 14:39:49.321 UTC [76] LOG:  database system was shut down at 2021-04-12 14:39:49 UTC
db_1           | 2021-04-12 14:39:49.344 UTC [1] LOG:  database system is ready to accept connections
```

On second not cleaned startup the following happens when I try to create a new battle:
```
db_1           | 2021-04-12 14:33:10.756 UTC [34] ERROR:  column "battle_id" does not exist at character 8
db_1           | 2021-04-12 14:33:10.756 UTC [34] HINT:  Perhaps you meant to reference the column "create_battle.battleid".
db_1           | 2021-04-12 14:33:10.756 UTC [34] STATEMENT:  SELECT battle_id FROM create_battle($1, $2, $3, $4);
thunderdome_1  | 2021/04/12 14:33:10 pq: column "battle_id" does not exist
```

My docker compose files looks as follows:
```
version: '3.1'

services:
  thunderdome:
    image: stevenweathers/thunderdome-planning-poker
    restart: always
    ports:
      - "8081:8080"
    depends_on:
      - db
    links:
      - db
    environment:
      APP_DOMAIN: ".127.0.0.1"
      COOKIE_SECURE: "false"
      DB_NAME: thunderdome
      DB_USER: thor
      DB_PASS: odinson
  db: 
    image: postgres:13.2
    restart: always
    environment:
      POSTGRES_DB: thunderdome
      POSTGRES_USER: thor
      POSTGRES_PASSWORD: odinson
    ports:
      - 5432:5432
    volumes:
      - C:/data/postgres:/var/lib/postgresql/data
```

**Comment by StevenWeathers** [comment]
Looks like the first issue is something surrounding postgres, I haven't seen that issue yet.
The second issue regarding creating a battle was just resolved with 1.6.1, and 1.6.2 resolves a battle deletion issue.
Please pull latest (1.6.2) and you should be stable again.

**Comment by snailcatcher** [comment]
Thanks @StevenWeathers!
Yes it works again as I would expect it.
